Texdiag/texdiag.cpp

```diff
@@ -1728,8 +1728,6 @@ namespace
 
                         if (bSigned)
                         {
-                        #pragma prefast(push)
```

Comment:
These need to stay for the codebase as I have internal tools that need this.

You can guard these with ``#ifdef _PREFAST_`` which won't be true for clang/LLVM.

    #ifdef _PREFAST_
    #pragma prefast(push)
    #pragma prefast(disable : 26453, "Shift here is never negative")
    #endif

Reply:
Note the change to `SIGN_EXTEND` above, PREFast no longer warns on these lines. FWIW my first attempt in 1b6517e generated 26450 instead ("operation causes overflow at compile time").

Reply:
Nice. The macro is definitely magic so I didn't have the time to figure that one out :)

Reply:
I've confirmed that with the ``SIGN_EXTENDED`` change in BC6BC7.cpp, the prefast suppression can also be removed.